styles/main.css

```diff
@@ -47,7 +37,7 @@ nav ul li a {
 #page-copyright{
 	color:#ffaa00;
 	text-transform: uppercase;
-	font-size: 3px;
+	letter-spacing: 3px;
```

Comment:
There is a `letter-spacing` CSS property that is different from `font-size`.  The former defines how much space is between each letter, the latter determines the size of each letter.